### PR TITLE
Fixed a bug when importing modules from pattern constructors

### PR DESCRIPTION
**Pull request by GearsDatapacks**
When #3418 was implemented, it looks like I overlooked a case:
```gleam
case option.None {
  option.Some(_) -> Nil
  // ^ Use "import module" code action here
  option.None -> Nil
}
```
Would become:
```gleam
import gleam/option

case option.None {
  option -> Nil
  option.None -> Nil
}
```

That is now fixed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -306,6 +306,10 @@
   inexhaustive case expressions matching on more than one subject.
   ([Surya Rose](https://github.com/GearsDatapacks))
 
+- Fixed a bug where the language server would remove the name of a constructor
+  pattern when importing an unknown module via a code action.
+  ([Surya Rose](https://github.com/GearsDatapacks))
```
Comment (lpil) [inline CHANGELOG.md:311]: I think this code action wasn't released yet, so we can skip this changelog entry

**Review by lpil — APPROVED** [review]
Thank you!!

File: CHANGELOG.md (hunk 1 above)
Comment (GearsDatapacks) [reply CHANGELOG.md:311]: Ah right, yes. That's still unreleased. I'm away at the moment so I'll have to wait until I get back to remove this and fix merge conflicts

**Comment by GearsDatapacks** [comment]
I have now removed the changelog entry.

**Review by lpil — APPROVED** [review]
Champ! Thank you

**Pull request merged**